Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml

```diff
@@ -111,15 +111,6 @@
     successChance: 0.5
     interactSuccessString: petting-success-slimes
     interactFailureString: petting-failure-generic
-  - type: ReplacementAccent
-    accent: slimes
```

Comment:
Dont remove lines, uncomment then add a note ``# Frontier`` so we can keep track of changes

Reply:
Thank you, bit silly of me to simply delete it outright, in hindsight.